typescript/cli/src/deploy/warp.ts

```diff
@@ -255,7 +257,7 @@ async function executeDeploy(params: DeployParams) {
 
   const [contractsFilePath, tokenConfigPath] = prepNewArtifactsFiles(outPath, [
     { filename: 'warp-route-deployment', description: 'Contract addresses' },
-    { filename: 'warp-ui-token-config', description: 'Warp UI token config' },
+    { filename: 'warp-config', description: 'Warp config' },
```

Comment:
why rename this? this shape is different from the warp deploy config

Reply:
yeah it's already `warp-config` on main, unsure why it's showing up as a change here